src/Runtime/EscaperRuntime.php

```diff
@@ -140,6 +140,10 @@ public function escape($string, string $strategy = 'html', ?string $charset = nu
             case 'html':
                 // see https://www.php.net/htmlspecialchars
 
+                if ('UTF-8' === $charset) {
+                    return htmlspecialchars($string, \ENT_QUOTES | \ENT_SUBSTITUTE, $charset);
```

Comment:
```suggestion
                    return htmlspecialchars($string, \ENT_QUOTES | \ENT_SUBSTITUTE, 'UTF-8');
```

Probably not faster, but maybe easier to read.